TACC/Frontera-Portal
# task/FP-690: Add Google Drive integration

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [FP-690](https://jira.tacc.utexas.edu/browse/FP-690)

## Summary of Changes: ##
Add the ability to connect a user account to Google Drive from the profile management page.

## Testing Steps: ##
1. Rebuild your image to pick up the new depenencies: `make build`
2. Run `./manage.py migrate` in `frontera_prtl_django`
2. Navigate to `/accounts/profile` and click "Connect to Google Drive" under "3rd Party Apps"
3. Check the privacy policy at https://cep.dev/googledrive-privacy-policy/

## UI Photos:
![image](https://user-images.githubusercontent.com/12601812/98984825-56ab1f80-24e8-11eb-946d-2f81d81526f6.png)
![image](https://user-images.githubusercontent.com/12601812/98984882-6460a500-24e8-11eb-942e-59e49bc54024.png)
![image](https://user-images.githubusercontent.com/12601812/98984947-7b9f9280-24e8-11eb-9527-a8ac350b1356.png)
